bundles/org.openhab.binding.broadlink/src/main/resources/OH-INF/thing/rm-types.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="broadlink"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="rm2">
+		<label>Broadlink RM2</label>
+		<description>The Broadlink RM 2/Pro is a Wi-Fi IR/RF transmitter with a temperature sensor.</description>
+		<channels>
+			<channel id="command" typeId="command"/>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="learningControl" typeId="learningControl"/>
+		</channels>
+		<representation-property>macAddress</representation-property>
+		<config-description-ref uri="thing-type:broadlink:rmconfig"/>
+	</thing-type>
+	<thing-type id="rm3">
+		<label>Broadlink RM3</label>
+		<description>The Broadlink RM 3/Mini is a Wi-Fi IR transmitter.</description>
+		<channels>
+			<channel id="command" typeId="command"/>
+			<channel id="learningControl" typeId="learningControl"/>
+		</channels>
+		<representation-property>macAddress</representation-property>
+		<config-description-ref uri="thing-type:broadlink:rmconfig"/>
+	</thing-type>
+	<thing-type id="rm3q">
+		<label>Broadlink RM3 v44057</label>
+		<description>The Broadlink RM 3/Mini is a Wi-Fi IR transmitter running v44057 firmware.</description>
```

Comment:
This sounds interesting, can you explain more as to why an exact firmware is mentioned in the description and label? The labels get used when items are auto created by `create equipement from thing` and this seems very odd to include the firmware here. Is there a good reason, if so please explain as any reviewer is going to ask this.

Reply:
It seems that there are 2 versions of the product with the same product name, but with a different firmware for different markets. The v44057 is a firmware specific to the Chinese market, but through grey imports have landed in the US / EU. However, in this market the same product with the same name, but with a different firmware is being sold. 

The v44057 firmware seems to have issues responding on status updates. Hence, a workaround was created.